students/Ben_freeman/lesson09/pngdiscover.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+big_list = []
+
+
+def dive(directory):
+    global big_list
+    contents = os.listdir(directory)
+    picture_holder = []
```

Comment:
It's ok to use _picture_holder_ if for some reason you want to know how many files you have found in this particular run, but you could just append files straight into _big_list_